tests/common/mod.rs

```diff
@@ -36,6 +45,33 @@ pub async fn setup_test_infrastructure(test_name: &str) -> (BitcoindNode, LdkNod
     (bitcoind, ldk1, ldk2)
 }
 
+// Sets up /tmp/lndk-tests folder where we'll store the bins, data directories, and logs needed
+// for our tests.
+//
+// The file tree structure looks like:
+//
+// /tmp/lndk-tests
+//             |
+//             +-- /bin (compiled lnd binary is stored here)
+//             |
+//             +-- /lnd-data (lnd data and logs are stored here)
+//             |
+//             +-- /ldk-data (ldk data and logs are stored here)
```

Comment:
Don't you think it would be easier to have these directories per-test rather than separate `lnd` and `lndk`? If you're debugging, you have to go look in two different places to figure out what's happened. 

`bin` dir works though!

Reply:
Oh yeah I like that idea, I'll switch that up